lib/vcap_env.rb

```diff
@@ -0,0 +1,13 @@
+require 'json'
+
+# Parses ENV['VCAP_SERVICES'] or similar JSON and sets ENV variables for user-provided:credentials
+module VcapEnv
+  def self.set_env(vcap_json)
+    vcap = JSON.parse(vcap_json)
+    env = vcap["user-provided"].find { |service| service["name"] == "compliance-viewer-env"  }
```

Comment:
That feels dirty to me too.  I keep going back to:
1) the README instructions specify "compliance-viewer-env" (maybe this is bad too)
2) if "compliance-viewer-env" isn't written here, it'll have to be written somewhere else -- potentially some other kind of settingsy thing?